docs/content/configuration/index.md

```diff
@@ -42,6 +42,9 @@ We recommend just setting the base ZK path and the ZK service host, but all ZK p
 |--------|-----------|-------|
 |`druid.zk.paths.base`|Base Zookeeper path.|`/druid`|
 |`druid.zk.service.host`|The ZooKeeper hosts to connect to. This is a REQUIRED property and therefore a host address must be supplied.|none|
+|`druid.zk.service.user`|The username to authenticate with ZooKeeper. This is an optional property.|none|
+|`druid.zk.service.pwd`|The password to authenticate with ZooKeeper. This is an optional property.|none|
```

Comment:
Would you please add an explanation that there are two options for configuring password and a link to `PasswordProvider`? http://druid.io/docs/latest/development/extensions-core/mysql.html#configuration can be a good reference.